src/Dialect/ONNX/ONNXShapeHelper.inc

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Reason to separate this file out?

Reply:
I don't like to have code in the header files, and by having it into an `inc` file, it looks a bit cleaner. I would like to have this templated code in the header files, as otherwise we need to put 50+ template instantiation, and every new op using that code pattern will have to do so also, which is a cause of pain as the errors for not doing so are cryptic.